_lifeofinternatpuppet/post_13.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: "Week 13"
+author: Disha-maker
+categories:
+  - team
+  - basics
+tags:
+  - week 13
```

Comment:
here you can add more tags: litmus_testing , concat, release maybe

Reply:
I was told by Norman to keep the tags consistent, rather than them being different each week :) So I am following that consistency :) Hope all of this is ok :)

Reply:
Okkk, that's good to know, thank you